jsparagus/emit/collect_handler_info/src/main.rs

```diff
@@ -21,7 +20,8 @@ fn get_method_names(mut in_file: File) -> Vec<String> {
                     if let syn::Visibility::Public(_) = method.vis {
                         if let syn::ReturnType::Type(_, t) = method.sig.output {
                             if let syn::Type::Path(path) = *t {
-                                if path.path.segments.first().unwrap().ident.to_string() == "Result" {
+                                if path.path.segments.first().unwrap().ident.to_string() == "Result"
+                                {
```

Comment:
this looks strange.
I guess reformatted later?

Reply:
This is the result of the reformat which happened after removing 2 lines from the file.